test/specs/collections/Table/TableHeader-test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import * as common from 'test/specs/commonTests'
+
+import TableHeader from 'src/collections/Table/TableHeader'
+
+describe('TableHeader', () => {
+  common.isConformant(TableHeader)
+  common.rendersChildren(TableHeader)
+
+  it('renders as a thead by default', () => {
+    shallow(<TableHeader />)
+      .should.have.tagName('thead')
+  })
+
+  describe('fullWidth', () => {
```

Comment:
There are two props with custom tests `fullWidth` and `singleLine`, we cannot use `_.kebabCase` because it will convert `singleLine` to `single-line` while it must be `single line`. `full-width` looks like design mistake in SUI, but we have that we have.

As really solution I see updating all common tests to handle additional argument, but I don't like it because we have only two cases for this now.

```diff
-export const propKeyOnlyToClassName = (Component, propKey, requiredProps = {}) => {
+export const propKeyOnlyToClassName = (Component, propKey, className, requiredProps = {}) => {
```